workforce-data-initiative/etp-api
# [#145043011] add -r tag for correct pip requirements installation

#### What does this PR do?
It edits the readme
#### Description of the task?
Correct the erroneous PIP installation command inside the python requirements section
#### How can this be manually tested?
You should be able to install requirements using the `pip install -r requirements.txt` command
#### Relevant Pivotal Tracker stories?
#145043011
